core/metacling/src/TCling.cxx

```diff
@@ -1101,7 +1101,7 @@ static void LoadCoreModules(cling::Interpreter &interp)
    clang::HeaderSearch &headerSearch = CI.getPreprocessor().getHeaderSearchInfo();
    clang::ModuleMap &moduleMap = headerSearch.getModuleMap();
    // List of core modules we need to load.
-   std::vector<std::string> neededCoreModuleNames = {"Core", "RIO"};
+   std::vector<std::string> neededCoreModuleNames = {"Core", "RIO", "stl", "libc"};
```

Comment:
Perhaps we should load them in the reverse order, i.e. starting from `libc`. However, we might want to start by loading `stl` first because that would avoid eager deserialization of lazy specializations.